block-padding/src/lib.rs

```diff
@@ -31,18 +31,21 @@ pub trait Padding {
 
     /// Pads message with length `pos` in the provided buffer.
     ///
-    /// `&buf[..pos]` is percieved as the message, buffer must contain at
-    /// least one block of leftover space, i.e. `buf.len() - pos >= block_size`
-    /// must be true. Otherwise method will return `PadError`.
+    /// `&buf[..pos]` is perceived as the message, the buffer must contain
+    /// enough leftover space for padding: if `pos % block_size == 0` then
+    /// a full extra block must be available, else `block_size - (pos %
+    /// block_size)` extra bytes must be available. Otherwise method will
```

Comment:
Aren't those two conditions fully equivalent to each other? You can change code accordingly.

Reply:
Yes, of course! I updated the code